libs/blocks/marquee/marquee.js

```diff
@@ -13,10 +13,12 @@
 /*
  * Marquee - v6.0
  */
+import { handleFocalpoint } from '../section-metadata/section-metadata.js';
```

Comment:
This will create performance problems and this should not be done.

Reply:
This would mean that section-metadata loads as a dependency in every marquee. This will impact LCP in a not insignificant way. It's bad enough we have the others... decorate, attributes, etc.

Reply:
I have tested the LCP score. It is not lowering the score, it remains the same. Also, @rgclayton suggested using the function if the same is present on section-metadata in one of my old PR. 
@rgclayton any suggestions for this?

Reply:
Can you share your draft docs that have your page speed? The ones in the PR are not sufficient.

Reply:
>Also, @rgclayton suggested using the function if the same is present on section-metadata in one of my old PR.
@suhjainadobe My suggestion was to treat `section-metadata` focal point the same as marquee, in that there isn't a new key value pair introduced/different authoring experience.

Reply:
We only have so much LCP budget. You're adding bytes to every single marquee request which is wrong. We should be gating the use of the module...

```js
async function decorateBackground(el) {
  if (el.children.length > 1) {
    const { default: initPos } = await import('./your-bg-handler.js');
    initPos(el);
  }
}
```
Something like this (it's definitely not exact) would be much more preferable.

Reply:
Actually, this particular function might be called more than one time depending on how many viewports are authored. So that would mean it will import the function multiple times. Can I directly write the function in the marquee rather than importing it from section-metadata.

Reply:
@auniverseaway I have pushed the changes. Could you please review the PR again?

Reply:
ESM de-dupes imports automatically. Also, if you build this correctly, you would intelligently figure out if the "row" of background has any positioning and import the function.

Reply:
@auniverseaway : It is done. Please review.